script/Arcueid/Day 12/12_00_ARC12_2.txt

```diff
@@ -216,42 +222,42 @@ Someone is coming.
 [sha:834af10323a0c8fa4a46f387b2c2c7b6a505027d]{
 -- Page 49, Offset 30638.
 -- 廊下の奥から、長い影を床に這わせて、
-From the depths of the hallway, a tall figure trudges forward,
+The figure emerging from the depths of the hallway casts a long shadow across the floor.
 }
 [sha:f2c46ab1e5ca51f3ab10545c2d830a4548496658]{
 -- Page 49, Offset 30639.
 -- かつん、かつん、と、冷徹な足音を響かせて。
-echoing the sound of cold footsteps.
+Their footsteps echo coldly as they advance.
 }
 [sha:46ea3225778d8e4eeb404eb996657075667a3a29]{
 -- Page 50, Offset 30640.
 -- ―――なにか、まずい。
-―――This is bad.
+―――Something's wrong here.
 }
 [sha:448958298102e9f0d339ca57a7700290ac1a5ec9]{
 -- Page 50, Offset 30641.
 -- ヴローヴのような、吹き荒ぶ脅威とは違う。
-This is different from the tempestuous threat of Vlov.
+The threat I sense now doesn't stem from the same sort of wild violence that drove Vlov.
 }
 [sha:4c05ea0702a58d403c4f7914c36985f01f360609]{
 -- Page 50, Offset 30642.
 -- <外界|そとがわ>からではなく<内界|うちがわ>から生じる潜在的な恐怖が、全身を脈打たせる。
-My entire body throbs from the latent fear, not coming from the outside, but from within.
+The fear that has set my entire body trembling isn't coming from outside, but from within.
```

Comment:
not a fan of "set my body trembling". Maybe "The latent fear sending tremors through my entire body"